planner/core/src/main/java/eu/seaclouds/platform/planner/core/DamGenerator.java

```diff
@@ -75,10 +74,22 @@
     public static final String SEACLOUDS_APPLICATION_INFORMATION_POLICY_TYPE = "seaclouds.policies.app.information";
     public static final String SEACLOUDS_APPLICATION_POLICY_NAME = "seaclouds.app.information";
     public static final String SEACLOUDS_NODE_PREFIX = "seaclouds.nodes";
+    public static final String SEACLOUDS_NODE_TYPES = "seaclouds-types";
+    public static final String SEACLOUDS_NODE_TYPES_VERSION = "0.8.0-SNAPSHOT";
 
     private static DeployerTypesResolver deployerTypesResolver;
     static Map<String, MonitoringInfo> monitoringInfoByApplication=new HashMap<>();
 
+    /**
+     *
+     * @param adp YAML string containing the ADP
```

Comment:
why a String? Can't you use the same approach used at https://github.com/SeaCloudsEU/SeaCloudsPlatform/blob/master/planner/matchmaker/src/main/java/eu/seaclouds/planner/matchmaker/Matchmaker.java#L172 ?

ADP is simply a TOSCA `ArchiveRoot`. wdyt?